packages/airnode-deployer/terraform/aws/modules/function/variables.tf

```diff
@@ -2,6 +2,12 @@ locals {
   uuid        = uuid()
   tmp_dir     = "/tmp/${var.name}#${local.uuid}"
   tmp_archive = "/tmp/${var.name}#${local.uuid}.zip"
+  # Read file `secrets.env` and return a list of tuples where first element is the key and the second is the value
```

Comment:
I spent some more time playing with `regexall` on https://terraform-online-console.com/. Here was a quick playlaod if you're interested:

```
regexall("(.+?)=(.*)", 
<<EOT
SECRET=value
SECRET=value= with=equals
abc=123
WORKS     tooo =     asd
a=1 b=2

MULTILINE=""

#abc=123
# abc=123
    abc=123
abc=1==23
abc=

         =       
EOT
)
```

I think it would be nice to add a comment here about what we consider valid (that's the most improtant). I think it should handle at comments and ignore trailing/leading spaces. 

Duplicates seem reasonable to allow (e.g. dotenv allows overrides), but you've mentioned TF doesn't handle that. That could be noted as well. We don't support multiline secrets, but that is not a common feature anyway (dotenv doesn't support it either).